config-connector/solutions/projects/kpt/shared-vpc/Kptfile

```diff
@@ -40,7 +40,7 @@ openAPI:
       x-k8s-cli:
         setter:
           name: org-id
-          value: ${ORG_ID?}
+          value: "${ORG_ID?}"
```

Comment:
This change is done automatically when the test-cli used `kpt cfg set` to reset the values.